src/server/lib/okta/idx/introspect.ts

```diff
@@ -5,6 +5,7 @@ import {
 	idxBaseResponseSchema,
 	idxFetch,
 } from './shared';
```

Comment:
This should be an absolute import.

Reply:
I'll make a separate PR to update any absolute imports!